src/session.rs

```diff
@@ -159,6 +160,16 @@ impl _Session {
                 Err(crate::ExtractError::Other(e)) => return Err(e),
                 _ => {}
             }
+            match kwargs.extract_item::<f64>("timeout") {
+                Ok(timeout) => {
+                    builder =
+                        builder.timeout(Duration::try_from_secs_f64(timeout).map_err(|_| {
+                            pyo3::exceptions::PyTypeError::new_err("Wrong timeout value")
```

Comment:
```suggestion
                            pyo3::exceptions::PyValueError::new_err("Wrong timeout value")
```

Reply:
Sorry, I was wrong, it should be a `ValueError`

Reply:
Make sense